ideas/307-breaking-whiisper.md

```diff
@@ -0,0 +1,131 @@
+---
+id: 307-breaking-whisper
+title: Breaking Whisper
+status: Draft
+created: 2018-09-24
+category: swarm / SITG experiment
+lead-contributor: corpetty
+contributors:
+exit-criteria: no
+success-metrics: yes
+clear-roles: yes
+future-iterations: yes
+---
+
+## Preamble
+    Idea: 307-breaking-whisper
+    Title: SITG Experiment #2: Breaking Whisper
+    Status: Draft
+    Created: 2018-09-24
+
+## Summary
+1) Understanding the limits of whisper (with and without optimizations)
```

Comment:
Is this to be filled out with more items? Authoritative recommendation for to do with it might be useful

Reply:
I don't understand what you mean here with the Authoritative recommendation part.

Reply:
As in: the paper stating:
- we studied this problem a lot
- here's our recommendation for people relying on Whisper (i.e. us)